data-loader/core/src/main/java/com/scalar/db/dataloader/core/dataexport/ExportManager.java

```diff
@@ -113,8 +116,11 @@ public ExportReport startExport(
           // TODO: handle this
         }
         processFooter(exportOptions, tableMetadata, bufferedWriter);
-      } catch (InterruptedException | IOException e) {
-        logger.error("Error during export: {}", e.getMessage());
+      } catch (InterruptedException
+          | IOException
+          | UnknownTransactionStatusException
+          | CrudException e) {
+        logger.error("Error during export: ", e);
```

Comment:
Not related to this PR, but `executorService` should be shut down in the `finally` block?

Reply:
@komamitsu san,
Thank you.
I will check this and I will create a PR to updated this if required after checking.